container/sagemaker/proxy_client.py

```diff
@@ -93,9 +124,14 @@ def _add_model_to_config_file(self, model_name, base_path, model_platform):
         config += '    name: "{}",\n'.format(model_name)
         config += '    base_path: "{}",\n'.format(base_path)
         config += '    model_platform: "{}"\n'.format(model_platform)
-        config += '  },\n'
+        config += '  }\n'
         config += '}\n'
 
         # write back to model-config.cfg
         with open(MODEL_CONFIG_FILE, 'w') as f:
             f.write(config)
+
+    def _delete_model_from_config_file(self, config_proto=None):
```

Comment:
why would this ever be called with None arg?

Reply:
Updated.